bgpd/bgp_route.c

```diff
@@ -4664,7 +4664,7 @@ void bgp_update(struct peer *peer, const struct prefix *p, uint32_t addpath_id,
 							 BGP_PATH_VALID);
 			}
 		} else {
-			if (accept_own)
+			if (safi == SAFI_MPLS_VPN && accept_own)
```

Comment:
btw, `accept_own` can't be true if it's not SAFI_MPLS_VPN (already validated in `bgp_accept_own()`.

Reply:
thanks.
I am closing the PR change, then.